docs/testing/README_TESTING.md

```diff
@@ -1,88 +1,30 @@
-# Testing WebApp - Overview of Process
+# Testing
 
-## Minimum Browsers
+[WebdriverIO](https://webdriver.io/) lets us automate our apps on the platforms provided by [BrowserStack](https://www.browserstack.com/).
 
-[Click here to see the minimum browser versions](https://docs.google.com/spreadsheets/d/1FlUMCvg1pNIO0IzJm0jQyvUW1YC_KHh-LO4l-OVIcog/edit#gid=1774503729) 
-that we support.
+## Config
 
-## How to test Wevote WebApp with BrowserStack
+Make a copy of `WebApp/tests/browserstack_automation/conf/browserstack.conf.template.js` and name the copy `WebApp/tests/browserstack_automation/conf/browserstack.conf.js` like so:
 
-If you haven't updated your dependencies in a while, run `npm install` from your terminal to install WebdriverIO (this is a framework that lets us test both the browser app and Cordova mobile apps with a single script). 
+```
+cd WebApp
 
-### Manual installation only
+cp tests/browserstack_automation/conf/browserstack.conf.template.js tests/browserstack_automation/conf/browserstack.conf.js
+```
 
-Copy `WebApp/tests/browserstack/browserstack.config-template.js` into `WebApp/tests/browserstack/browserstack.config.js`:
+Navigate to the [BrowserStack App Automate dashboard](https://app-automate.browserstack.com/dashboard/v2) to access your credentials under the "Access Key" menu and the app URLs `bs://ANOTHER-GENERATED-STRING-HERE` generated by uploading [the latest APK and IPA](https://drive.google.com/drive/u/0/folders/10tK7oqY7FKWhe0ilHDcli-DWpT9ldTFs) to BrowserStack. Adding these values to `browserstack.conf.js` allows us to test the provided apps.
 
-    (WebAppEnv) $ cd WebApp
-    (WebAppEnv) $ cp tests/browserstack/browserstack.config-template.js tests/browserstack/browserstack.config.js
+## [Page Object Pattern](https://webdriver.io/docs/pageobjects)
 
-### Automated installation start here
+This pattern allows us to abstract page information away from the tests. See the [`page_objects` directory](../../tests/browserstack_automation/page_objects/) for examples.
 
-You'll need to add your credentials to `browserstack.config.js`. Sign into Browserstack and navigate to the [BrowserStack Automate dashboard](https://automate.browserstack.com/). Press the down arrow next to where it says "Access Key" in the header. You should see your username ("YOUR-USERNAME" below) and access key ("ACCESS-KEY-HERE" below). You will need both of these values to upload the compiled App.
```

Comment:
Hi Antoinette, I can see moving these instructions for working with Browserstack (specifically testing the Apps) further down in this README file, but I think we should save it (as opposed to completely removing it).